examples/jsm/controls/ArcballControls.js

```diff
@@ -109,6 +109,8 @@ class ArcballControls extends EventDispatcher {
 
 		this._rotationAxis = new Vector3(); //axis for rotate operation
 
+		this._movement = new Vector3();
```

Comment:
It's actually better not to add helper variables to the object scope. Normally, we place them in module scope. I suggest you move `_movement` below the definition of `_offset`.

Sidenote: It would be good to avoid variables like `_v2_1` in object space, too. However, this detail is not relevant for this PR and not required to fix #22823.

Reply:
Done.